fuzzers/034-cmt-pll-pips/bits.dbf

```diff
@@ -1,7 +1,8 @@
-CMT_UPPER_T.CMT_TOP_R_UPPER_T_PLLE2_CLKFBIN.CMT_TOP_R_UPPER_T_PLLE2_CLK_FB_INT ^ CMT_UPPER_T.CMT_TOP_R_UPPER_T_PLLE2_CLKFBIN.CMT_TOP_R_UPPER_T_CLKFBIN
 CMT_UPPER_T.CMT_TOP_R_UPPER_T_PLLE2_CLKIN1.CMT_TOP_R_UPPER_T_PLLE2_CLK_IN1_INT ^ CMT_UPPER_T.CMT_TOP_R_UPPER_T_PLLE2_CLKIN1.CMT_TOP_R_UPPER_T_CLKIN1
 CMT_UPPER_T.CMT_TOP_R_UPPER_T_PLLE2_CLKIN2.CMT_TOP_R_UPPER_T_PLLE2_CLK_IN2_INT ^ CMT_UPPER_T.CMT_TOP_R_UPPER_T_PLLE2_CLKIN2.CMT_TOP_R_UPPER_T_CLKIN2
 
 CMT_UPPER_T.CMT_TOP_R_UPPER_T_PLLE2_CLKIN1.CMT_TOP_L_UPPER_T_PLLE2_CLK_IN1_INT ^ CMT_UPPER_T.CMT_TOP_R_UPPER_T_PLLE2_CLKIN1.CMT_TOP_L_UPPER_T_CLKIN1
 CMT_UPPER_T.CMT_TOP_R_UPPER_T_PLLE2_CLKIN2.CMT_TOP_L_UPPER_T_PLLE2_CLK_IN2_INT ^ CMT_UPPER_T.CMT_TOP_R_UPPER_T_PLLE2_CLKIN2.CMT_TOP_L_UPPER_T_CLKIN2
 
+28_43 29_42 29_43 28_44
```

Comment:
Can this bit group be expressed using tags?

Reply:
For now I think it cannot. The `dbfixup.py` would need modification to be able to interpret a group of tags instead of bits.

Reply:
`dbfixup.py` supports that?  If it is based around a zero feature, you can do <tag list>, <zero tag> and I believe `dbfixup.py` has support for multiple `^` tags.

As part of this PR, can you double check that `dbfixup.py` doesn't already have what you need, and if not propose something that could work?

Reply:
But in this case there is no all zero feature. What I need is specification of a group of bits common to a group of features so that if a bit from that group is not set in a feature, then it should be added there as negatad.

I've been through the code of `dbfixup.py` and I didn't see a way to specify that trough feature names, you can only do that providing bit names. But I can add a way to make it possible.

Reply:
> But I can add a way to make it possible.

Let's explore that, but after we get the CI green.